app/assets/javascripts/models/story.js

```diff
@@ -117,7 +117,7 @@ Fulcrum.Story = Backbone.Model.extend({
 
   estimated: function() {
     var estimate = this.get('estimate');
-    return !(estimate === undefined || estimate === null);
+    return !(estimate === undefined || _.isNull(estimate));
```

Comment:
I was hoping that underscore would provide a method for that. Unfortunately, [_.isUndefined](http://underscorejs.org/#isUndefined) works for values and not for objects.